src/Migration.php

```diff
@@ -78,6 +78,22 @@ class Migration extends Expression
     /** @var array use this array in extended classes to overwrite or extend values of default mapping */
     public $mapToAgile = [];
 
+    /** @var array stores migrator to use based on driver */
+    private static $registry = [
```

Comment:
Can this be protected static property not private? I think that helps in case you want to extend this class.

Reply:
I intentionally set it to `private`.
The idea is that if you call generic class `Migration::of` the migrator class will be resolved based on driver. (the `registry` property is visible)
But if you call specific migrator class e.g `Migration\MySQL::of` it will not be resolved (the `registry` property is NOT visible).
MySQL migrator will be used explicitly.

This functionality is used in the `MigratorConsole` if `migrator_class` is set explicitly.

Reply:
Cool. Maybe you can add this kind of explanation/comment in there is sourcecode too. Otherwise I'm more than sure that at some point somebody will try to change it to `protected` and fail :)